df.history.xml

```diff
@@ -778,6 +778,15 @@
         <int32_t init-value='-1'/>
     </struct-type>
 
+    <enum-type type-name='identity_type' base-type='int32_t'>
+         <enum-item name='Hiding_Curse' comment="Inferred from Units.cpp after examining code using 'unk_4c'"/>
+         <enum-item name='Unk_1' comment="Not seen. A guess is that it might have been used by deities under cover"/>
+         <enum-item name='True_Name' comment="E.g. of demonic overlords. Can be used by adventurers to gain sway over them"/>
+         <enum-item name='False_Identity' comment="For underhanded purposes"/>
+         <enum-item name='Unk_4'/>
+         <enum-item name='Identity' comment="Claim a new official identity, seen when religious appointments are received"/>
```

Comment:
I would probably get rid of the underscores in the names, just for convention's sake. But since it's going to be used in 3 PRs, I'll defer to your decision.

Reply:
I decided to wait with exportlegends until acceptance was imminent as there might still be comments to deal with (like the one above), as well as the possibility of unrelated changes to exportlegends (e.g. a possible export to support religion relations, as discussed at the site).

While I dislike mashing words together unless it's part of the (human) language's conventions, I can accept it as long as it's not those awful camels: they're extremely ugly in my view, so I'll do the changes in the next few hours.